.circleci/config.yml

```diff
@@ -35,6 +35,7 @@ commands:
             cd geth*/
             install -m 755 geth ~/.local/bin/geth
             pip install -U poetry
```

Comment:
Is this `pip` call still required if we're going with the script way?

Reply:
No it isn't. I removed that line.